lib/MiniDroneBtAdapter.js

```diff
@@ -403,7 +403,10 @@ class MiniDroneBtAdapter extends EventEmitter {
         const manufacturer = peripheral.advertisement.manufacturerData;
         const matchesFilter = localName === this.options.droneFilter;
 
-        const localNameMatch = matchesFilter || DRONE_PREFIXES.some((prefix) => localName.indexOf(prefix) >= 0);
```

Comment:
localName was undefined at this point, but by defaulting localNameMatch to false it still caught it (presumably via the manufacturer match). I can print out the exact object when I get home if that'd help.